python/cuml/preprocessing/label.py

```diff
@@ -226,10 +225,16 @@ def inverse_transform(self, y, threshold=None):
         arr : array with original labels
         """
 
+        if has_scipy():
+            from scipy.sparse import isspmatrix as scipy_sparse_isspmatrix
+        else:
+            def scipy_sparse_isspmatrix(x):
```

Comment:
Since you have to define this function many times and a similar pattern may be used for other import issues, what do you think about adding a dummy function in cuml.utils.import_utils that returns False?

 Then you would do:
 ```
        if has_scipy():
            from scipy.sparse import isspmatrix as scipy_sparse_isspmatrix
        else:
            from cuml.utils.import_utils import dummy_function as scipy_sparse_isspmatrix
```
?

The symmetry here looks nicer to me but maybe it's just my style...

Reply:
Done.